product_hunt_api/product_hunt_api/actions.py

```diff
@@ -0,0 +1,62 @@
+import requests
+import json
+from django.http import HttpResponse
+from .constants import PRODUCT_HUNT_ENDPOINT, HEADERS
+
+
+# this functions make request to product hunt api and fetch the data
+
+def make_request_to_product_hunt():
+
+    json_data = requests.get(PRODUCT_HUNT_ENDPOINT, headers=HEADERS)
```

Comment:
Wouldn't this fetch all records, you should paginate, take input from user for this